middle_end/flambda2/simplify/simplify_apply_expr.ml

```diff
@@ -672,8 +672,11 @@ let simplify_direct_partial_application ~simplify_expr dacc apply
                ~function_relative_history:Inlining_history.Relative.empty ~dbg
                ~name
         in
+        (* CR sspies: The debuginfo [dbg] here is only a guess. See whether it
+           works and confirm it's the right one. It is passed below to the code
+           create, so this is promising. *)
```

Comment:
We are in the case of a partial application of a function, and we generate a wrapper. The debuginfo we currently put is the one of the application, because we don't really have a better location to put there, so I think this is good too.